index.html

```diff
@@ -54,8 +54,8 @@ <h4 class="post-title"><strong>{{ post.title }}</strong></h4>
             <h3 class = "welcome"> Welcome </h3>
             <p class = "welcome">
                     Welcome to the University <br>of <b>Calgary Competitive Programming Club</b> website. 
-                    We meet every Wednesday night from 6:00-7:30pm in MS 160 for coaching and practice. 
-                    Saturday practice contests are also often held. To join the club, just join our <u>mailing list</u>.
+                    We meet every Wednesday night from 6:00-7:30pm on <a href="https://discord.gg/bFk6kKf">Discord</a> for coaching, practice and mingling. 
+					Join our Discord to hang out, learn some stuff and meeting new people!

```

Comment:
Change to 'Join our Discord to gain access to free weekly algorithmic topic sessions, get problem solving help, and meet new people!'